src/Symfony/Component/Messenger/Tests/Fixtures/MessageHandlerFailingFirstTimes.php

```diff
@@ -0,0 +1,39 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\Messenger\Tests\Fixtures;
+
+class MessageHandlerFailingFirstTimes
```

Comment:
Can we call this something else - like prefix with test? I don't want it to pollute auto-complete for `MessageHandlerInterface`

Reply:
Good point 👍